Dockerfile

```diff
@@ -12,15 +12,14 @@ ADD setup.py /opt/atomicapp/
 ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
-
-RUN pip install .
+RUN python setup.py install
 
 WORKDIR /atomicapp
 VOLUME /atomicapp
-
-LABEL RUN docker run -it --rm --privileged --net=host -v `pwd`:/atomicapp -v /run:/run -v /:/host --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE -v run /atomicapp
+ 
+LABEL RUN docker run -it --rm --privileged --net=host -v `pwd`:/atomicapp -v /run:/run -v /:/host --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE -v RUNARGS run /atomicapp
```

Comment:
if `RUNARGS` is in both cases just `-d` we can move it into the Dockerfile itself, as it is static input